README.md

```diff
@@ -1,13 +1,9 @@
 # Project Name
```

Comment:
Hi Noemi! happy to be reviewing your nice and neat code! 😀 🌈
I would add the name of the project here in the Readme section so that everyone can read it.
Project name > Guess who game